components/dashboard/src/index.css

```diff
@@ -59,7 +59,7 @@
     }
 
     a.gp-link {
-        @apply underline underline-thickness-thin underline-offset-small text-gray-400 dark:text-gray-600 hover:text-gray-500 dark:hover:text-gray-500;
+        @apply text-blue-500 dark:text-gray-600 hover:text-gray-500 dark:hover:text-gray-500;
```

Comment:
I assumed you would want to have that streamlined across the product?

Reply:
> I assumed you would want to have that streamlined across the product?

That would be ideal, right? That's why `gp-link` class was added in https://github.com/gitpod-io/gitpod/pull/5112/commits/2098802a446dbb3d1c245719fa555188ad49bf0b.

In the future we could have links that use a different style but let's try to keep one style for the text links that could later become a link component. ⚫ ⚫ ⚫ 

We could also update text links to use the blue color as the product evolves since this could probably be more appropriate but bringing this up as updating the text link style here would update the help link style in quite a few places that we may also need to check in terms of accessibility and visual design. I'd suggest to discuss this in a follow up issue. 🍔

Reply:
so you want me to roll back the change and only make this one link blue?

Reply:
Oh, sorry if the message was confusing! 🙈 I was suggesting to keeping the underline style everywhere and update it later but just checked how the blue link renders across the product and looks great. 🔮 

**suggestion**: LET'S KEEP IT BLUE. 🔵 

**issue**: However, we still need to have proper blue hover states and good blue colors for the dark theme. Give me 15 min to get back to this. ⏱️

Reply:
Ready! We can think about underline text decoration or improving text link accessibility in the future.

```suggestion
        @apply text-blue-500 hover:text-blue-600  dark:text-blue-400 dark:hover:text-gray-500;
```

Reply:
**nitpick(non-blocking)**: Oops! Maybe the screenshots in https://github.com/gitpod-io/gitpod/pull/5555#discussion_r715605928 caused some confusion on the help link colors but the code suggestion above includes the correct help link colors for dark layouts. Blue on light layout and lighter blue on dark layout. Can open a follow up PR for this if needed.